index.html

```diff
@@ -65,12 +65,12 @@
     <div class=feature>
       <i class="fa fa-check"></i>
       <h3>Simplicity</h3>
-      <p>We like to keep it simple and intuitive, with only the features you use</p>
+      <p>We like to keep it simple and intuitive, with only the features you use.</p>
```

Comment:
We keep it simple and intuitive. 

But that doesn't look as nice visually on the page :wink: